src/components/MapDetail/components/Plot/Plot.js

```diff
@@ -1,94 +1,169 @@
 import React from 'react'
 import { Box, Text } from 'grommet'
-import { ResponsiveLine } from '@nivo/line'
-import { string } from 'prop-types'
-import getAverages from 'helpers/getAverages'
+import { ResponsiveScatterPlot } from '@nivo/scatterplot'
+import { line } from 'd3-shape'
```

Comment:
This works great, just confirming there's nothing in `nivo` that can do the same? I checked out `nivo` a little and don't think there is, and since it's built on `d3` this makes perfect sense, just checking.

Reply:
Agreed! When I searched for how to render multiple layers on a `nivo` plot, their suggested example was using d3-shape, and I'll definitely see if the line layer can be interactive for changing the selected year, too.